sympy/core/tests/test_args.py

```diff
@@ -95,7 +95,6 @@ def test_sympy__assumptions__sathandlers__UnevaluatedOnFree():
     from sympy import Q
     assert _test_args(UnevaluatedOnFree(Q.positive))
     assert _test_args(UnevaluatedOnFree(Q.positive(x)))
-    assert _test_args(UnevaluatedOnFree(Q.positive(x*y)))
```

Comment:
Was this dropped accidentally?

Reply:
I think I might have deleted it while tracking `_test_args`. I will fix it.

Reply:
I should always look for changes before commiting. I just ran tests and pushed it.